srcalc/src/main/java/gov/va/med/srcalc/service/DefaultAdminService.java

```diff
@@ -107,4 +107,30 @@ public void saveVariable(final AbstractVariable variable)
         fLogger.debug("Getting all RiskModels.");
         return fRiskModelDao.getAllRiskModels();
     }
+    
+    @Override
+    @Transactional(readOnly = true)
+    public RiskModel getRiskModelForId(final int modelId)
+    {        
+        fLogger.debug("Getting RiskModel for {}.", modelId);
+        return fRiskModelDao.getRiskModelForId( modelId );
+    }
+
+    /**
+     * Saves the {@link RiskModel} 
+     */
+    @Override
+    @Transactional
+    public void saveRiskModel( final RiskModel model ) 
+    {    	
+        if( getRiskModelForId( model.getId() ) == null ) 
```

Comment:
Shouldn't happen since there is currently no way for the user create or add new Ids.
Its just a sanity check / warning.